src/includes/performance/configure-sample-rate/java.spring-boot.mdx

```diff
@@ -17,6 +17,7 @@ import org.springframework.stereotype.Component;
 class CustomTracesSamplerCallback implements TracesSamplerCallback {
   @Override
   public Double sample(SamplingContext samplingContext) {
+    HttpServletRequest request = (HttpServletRequest) context.getCustomSamplingContext().get("request")
```

Comment:
the kotlin code snippet assumes `samplingContext.customSamplingContext` is nullable and check for it, here we don't, if its a @Nullable field, lets do NPE check here too